src/ucp/proto/proto.h

```diff
@@ -141,6 +141,9 @@ typedef struct {
 
     /* Map of used lanes */
     ucp_lane_map_t lane_map;
+
+    /* Selected count */
+    size_t         selected_count;
```

Comment:
Please use `selections` instead of `selected` since this field represents the number of the protocol selections.

Reply:
sure